src/arm/api.h

```diff
@@ -43,6 +43,7 @@ enum cpuinfo_arm_chipset_series {
 	cpuinfo_arm_chipset_series_hisilicon_k3v,
 	cpuinfo_arm_chipset_series_hisilicon_hi,
 	cpuinfo_arm_chipset_series_hisilicon_kirin,
+	cpuinfo_arm_chipset_series_hisilicon_tsv100,
```

Comment:
This value is unused, please remove it. On Android we detect SoC name is necessary to detect cache size, so we parse it from system properties into this enum. For Kunpeng cache size seems to be uniquely identified by MIDR, so SoC identification is unnecessary.

Reply:
remove it